modules/dataframe/include/inviwo/dataframe/datastructures/dataframe.h

```diff
@@ -255,13 +266,14 @@ struct DataTraits<DataFrame> {
         tb(H("Number of Rows: "), rowCount);
 
         utildoc::TableBuilder tb2(doc.handle(), P::end());
-        tb2(H("Col"), H("Format"), H("Rows"), H("Name"), H("Column Range"));
+        tb2(H("Col"), H("Format"), H("Rows"), H("Name"), H("Column Range"), H("Min"), H("Max"),
+            H("Unit"));
         // abbreviate list of columns if there are more than 20
         const size_t ncols = (data.getNumberOfColumns() > 20) ? 10 : data.getNumberOfColumns();
 
         auto range = [](const Column* col) {
-            if (col->getRange()) {
-                return toString(*col->getRange());
+            if (col->getCustomRange()) {
+                return toString(*col->getCustomRange());
             } else {
                 return std::string("-");
```

Comment:
Hm, should we instead just always use `getRange()` here? The main reason for the if statement was the optional range.

Reply:
think we also print the min max though